### DISCUSSION TITLE
2024-11-07 Meeting Notes

### DISCUSSION THREAD

**Discussion by joshcullen** [joshcullen/CEG_operationalization#7] [General]
## Summary

Reviewed what we talk about in previous meeting and next steps to take for next week. Emphasis on developing pipeline (datasets, code, tools) specifically for leatherback models for now. Can expand for other operational tools later.

## Attendees

* Josh Cullen (@joshcullen)
* Heather Welch (@HeatherWelch)

## Notes

- Reviewed what we discussed from previous meeting
- Josh went through the different tools for communicating and project mgmt in repo
- Discussed some of the updates Josh made to the utility functions Heather added

## To-Do

- Heather will create metadata spreadsheet that describes each of the remote sensing products and variables of interest
- Josh will update utility functions and make them more consistent (for I/O)

**Comment by HeatherWelch** [comment]
## Added model_metadata.csv to /model_prediction
I was imaging this file would contain model metadata across all the species models, so that we can just refer to one file to have all the relevant information. I included metadata that we'll need for acquisition and prediction (does it belong here, or somewhere else?). Also the two leatherback models pull from ROMS and CMEMS, so we can set ERDDAP aside for now.

Column names
_product_: product name. Doesn't really have meaning for ROMS. There's an "Other" category for covariates that we don't get from CMEMS, ERDDAP, or ROMS (e.g. for bathymetry)
_variable_: variable name within the raw ncdf files
_url_: blank, only meaningful for ERDDAP
_data_type_: ROMS, CMEMS, ERDDAP, Other (e.g. for bathymetry)
_model_var_name_: Name of the variable in the model object (for prediction)
_category_: raw - means directly access from the netcdf, even though there might still be some post processing like resolution change or log (e.g. sst); derived - means derived from another variable on this sheet (e.g. sst_sd)
_model_: name of the model using this data. We'll have to think about how to handle it when multiple models use the same data source. I think it would probably be its own row, as there will almost certainly be different post-processing

**Reply by joshcullen** [reply]
Sounds good. I built this repo using a NMFS Open Science template, but it just had a bunch of extra stuff that I don't think was useful (at least for now). So I deleted those extra files/folders and moved the metadata CSV into a "docs" folder at the root of the directory
